### PR TITLE
Add go modules and refactor packages

### PR DESCRIPTION
**Pull request by gordonbondon**

### CODE REVIEW

**Comment by hatemosphere** [comment]
@gordonbondon can you please divide the actual code changes and go mod introduction into two different commits to simplify the review? :trollface:

File: cmd/cmd.go [hunk 1]
```diff
@@ -0,0 +1,62 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
```
Comment (hatemosphere) [inline cmd/cmd.go]: what are your thoughts about cobra vs kingpin?

Comment (gordonbondon) [reply cmd/cmd.go]: https://github.com/alecthomas/kingpin last commit 9 month ago, either it's perfect or dead. I'll check if we can use it.

**Comment by gordonbondon** [comment]
@hatemosphere changes were split into two commits

**Comment by alexandrst88** [comment]
thanks @gordonbondon 🤗

**Pull request merged**